spi/src/main/java/org/jboss/pnc/spi/executor/BuildExecutionConfiguration.java

```diff
@@ -51,7 +55,10 @@
     SystemImageType getSystemImageType();
 
     boolean isPodKeptOnFailure();
-    
+
+    @Override
```

Comment:
Added it in BuildExecution since it was moved there and I forgot to remove the method signature from the BuildExecutionConfiguration interface.